openedx/core/djangoapps/credentials/tasks/v1/tasks.py

```diff
@@ -70,35 +77,26 @@ def send_grade_to_credentials(
         percent_grade (float): Number representing the learner's grade in this course run
         grade_last_updated (string): String describing the last time this grade was modified in the LMS
     """
-    logger.info(f"Running task send_grade_to_credentials for username {username} and course {course_run_key}")
+    data = {
+        'username': username,
+        'course_run': course_run_key,
+        'letter_grade': letter_grade,
+        'percent_grade': percent_grade,
+        'verified': verified,
+        'lms_last_updated_at': grade_last_updated
+    }
+    logger.info(f"Running task `send_grade_to_credentials` for username {username} with data: {data}")
```

Comment:
[question] Is this considered safe information for our logs? it's not exactly PII – it's just a username and grades, nothing personally identifiable – but I wanted to make sure we consider that a safe update.

to be clear, I'm not saying it's not. I just assume we have standards about what you can send the logs and what you can't,  & I don't know them.

Reply:
This is one of those "stuck between a rock and a hard place" logging decisions for me. I'm trying to value what would be valuable for debugging, balanced against what potential PII are we logging. Over the years, we've tried really hard NOT to log username (and log LMS User Id instead as the identifier in logs)... but Credentials mostly cares about `username`. 

I thought the value of having some insight as to what we're sending to Credentials is more valuable than keeping the `username` out of our logs. AFAIK it's a strong preference, not a hard requirement to exclude info like `username` (which I *do* think is considered PII).

Reply:
yeah, that makes sense. I was more worried about sending the username with the grade, but that's not really PII by any definition, even if it is something the user would expect to have extra privacy applied to. :shipit: